config/race_test.go

```diff
@@ -0,0 +1,75 @@
+package config
+
+import (
+	"math/rand"
+	"testing"
+	"time"
+)
+
+func TestIntRace(t *testing.T) {
```

Comment:
These aren't real tests since they have no assertions. They are supposed to just prove that one has a data race and the other version does not. So they are meant be executed with `-race` which is going to make `TestIntRace` fail.